.gitignore

```diff
@@ -126,9 +127,6 @@ $tf/
 # Guidance Automation Toolkit
 *.gpState
 
-# Visual Studio Code
-.vscode
```

Comment:
Are we sure we want to include the `.vscode` files? I've seen very few GitHub projects that check these in. Looking at `extensions.json` and `settings.json`, I'm not sure I'd use most of these settings — Prettier being the exception. 

What do you think about leaving these out to keep the repo clean? Open to hearing your thoughts! 😊

Reply:
for sure, most projects keep .vscode improving the developer experience when using vscode - it also makes the onboarding easier. (dot files and folders can be hidden en most editors if needed)

Reply:
In my experience, the `.vscode` folder often accumulates various files that differ between Windows and Mac. For example, `settings.json`, `launch.json`, `tasks.json`, etc., probably shouldn't be included in the repo.

I tested checking out the solution, deleting these files, and was still able to run the frontend without issues.

How about we stick to git-ignoring `.vscode` and only explicitly check in the `extensions.json` for recommendations? And perhaps we could trim down those recommendations to the absolute essentials? I'd prefer not to have a bunch of extensions I won't use. 😊